Symfony/CS/ConfigInterface.php

```diff
@@ -105,4 +105,20 @@ public function usingCache();
      * @return bool
      */
     public function usingLinter();
+
+    /**
+     * Sets the path to the cache file.
+     *
+     * @param string $cacheFile The path to the cache file
+     *
+     * @return ConfigInterface The same instance
```

Comment:
Also, the `addCustomFixer` method isn't chainable. Why should this one be chainable, or should they both be changed?